catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -160,16 +167,11 @@ private void apply(File file) throws Exception {
 
     Outer outer;
     try {
-      outer = objectMapper.readValue(data, Outer.class);
+      outer = GSON.fromJson(data, Outer.class);
```

Comment:
✏️ We were forced to read streams in a string with Boon because of a bug. Should we take advantage of Gson and read directly from it?

Reply:
Same regarding logging the json string.